src/AspNet.Security.OAuth.Line/LineAuthenticationOptions.cs

```diff
@@ -0,0 +1,50 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using static AspNet.Security.OAuth.Line.LineAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.Line
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="LineAuthenticationHandler"/>.
+    /// </summary>
+    public class LineAuthenticationOptions : OAuthOptions
+    {
+        public LineAuthenticationOptions()
+        {
+            ClaimsIssuer = LineAuthenticationDefaults.Issuer;
+            CallbackPath = LineAuthenticationDefaults.CallbackPath;
+
+            AuthorizationEndpoint = LineAuthenticationDefaults.AuthorizationEndpoint;
+            TokenEndpoint = LineAuthenticationDefaults.TokenEndpoint;
+            UserInformationEndpoint = LineAuthenticationDefaults.UserInformationEndpoint;
+
+            Scope.Add("profile");
+            Scope.Add("openid");
+            Scope.Add("email");
```

Comment:
Could use the constant as mentioned in the handler comments.

Reply:
Does this need to be the default, and instead something that's opted into?

Reply:
I update it and see above. Thank you for your suggestions.

Reply:
By the way, how can I make a version to target netcoreapp3.1(use the branch rel/3.1.7)?